azurelinuxagent/ga/cgroupapi.py

```diff
@@ -158,6 +158,31 @@ def get_current_cpu_quota(unit_name):
             log_cgroup_warning("Error parsing current CPUQuotaPerSecUSec: {0}".format(ustr(e)))
             return "unknown"
 
+    @staticmethod
+    def get_current_memory_quota(unit_name):
+        """
+        Returns the memory quota for the given unit in bytes, or 'infinity' or 'unknown' if not set.
+        """
+        try:
+            mem_quota = systemd.get_unit_property(unit_name, "MemoryHigh").strip().lower()
+            if mem_quota == "infinity":
+                return mem_quota  # No limit on memory usage
+            elif mem_quota.endswith("k"):
```

Comment:
is this case insensitive? If not, should the check be case insensitive or will unit always be lowercase?